_episodes/09-branches.md

```diff
@@ -171,87 +139,79 @@ $ git checkout experiment  # switch to branch "experiment"
 $ git branch               # list all local branches and show on which branch we are
 ```
 
-- The branch is created from the `master` branch.
-- We commit our changes to this branch. 
-- Stage this and commit it with the message "let us try with some cilantro"
+- Verify that you are on the `experiment` branch:
```

Comment:
and note that `git graph` also makes it clear what branch you are on: `HEAD -> branchname`.